terra

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
+
```

Comment:
Did you mean to commit this file and the terra-cli.tar file? I think these are both generated files.